### ISSUE TITLE
Can't install 1.5.8 on Fedora

### ISSUE THREAD

**Issue by ghost** [bisq-network/bisq#5266]
### Description

The latest version `rpm` is broken?

#### Version

1.5.8

### Steps to reproduce

```shell
wget https://github.com/bisq-network/bisq/releases/download/v1.5.8/Bisq-64bit-1.5.8.rpm
sudo dnf install Bisq-64bit-1.5.8.rpm
```

### Expected behaviour

Bisq can be run.

### Actual behaviour

No Bisq installed.

### Screenshots

```shell
> locate bisq|rg -v "\.local"
/usr/share/man/mann/tk_bisque.n.gz
```

#### Device or machine

```shell
> lsb_release -a
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: Fedora
Description:    Fedora release 33 (Thirty Three)
Release:        33
Codename:       ThirtyThree
```

**Comment by ghost** [comment]
Actually it's installed

```shell
> ll /opt/Bisq/
total 728K
drwxr-xr-x. 2 root root 4.0K Mar  2 17:59 app/
-rwxr-xr-x. 1 root root  14K Mar  1 14:23 Bisq*
-rw-r--r--. 1 root root  172 Mar  1 14:23 Bisq.desktop
-rwxr-xr-x. 1 root root  46K Mar  1 14:23 Bisq.png*
-rw-r--r--. 1 root root 648K Mar  1 14:23 libpackager.so
drwxr-xr-x. 2 root root 4.0K Mar  2 17:59 resources/
drwxr-xr-x. 6 root root 4.0K Mar  2 17:59 runtime/
```

but it's not recognizable by Gnome search (like it was for `1.5.6`)

**Comment by ripcurlx** [comment]
I'm not aware of any changes in .rpm generation. But I personally only tested the .deb bundle. There will be some changes with the new build pipeline in #4242 we'll switch to in the next or the release after, so maybe this fixes your issue as well.

**Comment by ripcurlx** [comment]
Closed it by mistake.

**Comment by ghost** [comment]
Good to know, I'll check it when it's ready. Thanks!

**Comment by cd2357** [comment]
This should be fixed in the most recent release (v1.6.4). This already includes the changes mentioned above for the .rpm generation.

Please try it out and let us know if it's still an issue for you.

**Comment by ghost** [comment]
`v1.6.4` works, thanks!
